packages/syft/src/syft/core/node/common/client.py

```diff
@@ -294,6 +294,14 @@ def send_eventual_msg_without_reply(
             msg=signed_msg, timeout=timeout
         )
 
+    def url_from_path(self, path: str) -> str:
```

Comment:
MAYBE: could we rename it to `modify_url_to_client_host` or any other naming, and rename `path` to `url`